src/App.tsx

```diff
@@ -15,71 +16,122 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type BtnProps = {
+  dataCy: string,
+  content: string,
+  classes?: string[],
+  handler: () => void
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const GoodsButton: React.FC<BtnProps> = (
```

Comment:
imho component goodsBtn and type goodsBtn its extra code and you dont need it

Reply:
but I like it :)